tools/KodKod.Analyzer/KodKod.Analyzer/ApiActionsAreAttributeRoutedAnalyzer.cs

```diff
@@ -0,0 +1,40 @@
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace KodKod.Analyzer
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    public class ApiActionsAreAttributeRoutedAnalyzer : KodKodAnalyzerAnalyzerBase
```

Comment:
Presumably we could put a couple of fixers - that recommend decorating it with `HttpGet` \ `HttpPost`.